config/config.go

```diff
@@ -83,6 +83,7 @@ type Hybrik struct {
 	AuthKey        string `envconfig:"HYBRIK_AUTH_KEY"`
 	AuthSecret     string `envconfig:"HYBRIK_AUTH_SECRET"`
 	Destination    string `envconfig:"HYBRIK_DESTINATION"`
+	PresetPath     string `envconfig:"HYBRIK_PRESET_PATH"`
```

Comment:
You can add the struct tag ``default:"transcoding-api-presets"`` and drop [this if](https://github.com/NYTimes/video-transcoding-api/pull/185/files/b3e136a57be375baaccba9734867a9edcc153211..0641150ac2fc07ef3b4176602b500ed04a7ee061#diff-87bbf9707c283d056c8721f73fa58e78R419).